doc/index.rst

```diff
@@ -390,6 +390,10 @@ In such cases, you can specify the script(s) in `BEFORE_SCRIPT` and/or `AFTER_SC
   script:
     - .ci_config/travis.sh
 
+Multiple scripts/commands can be passed using semi-colon as a delimitter, e.g.::
+
+    - BEFORE_SCRIPT="echo 'Here is command 1.';echo 'Here is command 2.';"
```

Comment:
There are two possible issues with this example:
1. This first command (in general) might fail without failing the test. To it is better to use `command1 && command2`.
2. If you use double quotes at the outer level, variables will get expanded right away. 

`BEFORE_SCRIPT='echo "Here is command 1." && echo "Here is command 2."'`

Reply:
Those are generic `bash` tips but very valuable in this context. I decided to avoid adding those and instead just mention that the values for the variables in question follow general bash grammar.

Reply:
The 2nd hint does not cover bash subsitution, but travis substitution:
`BEFORE_SCRIPT="echo $(pwd)"` will print the (outer) travis directory.
`BEFORE_SCRIPT='echo $(pwd)'` will print the (inner) docker directory.